python/ray/serve/tests/test_config_files/pid_more_replicas.yaml

```diff
@@ -0,0 +1,8 @@
+import_path: ray.serve.tests.test_config_files.pid.node
+
+deployments:
+
+- name: f
+  num_replicas: 2
```

Comment:
Could we also add checks for the other two config update options (`user_config` and `autoscaling_config`)? It would help prevent regressions in their implementations in the future.